### ISSUE TITLE
User required to press a key multiple times to get to the next bash command

### ISSUE THREAD

**Issue by jasonmesser7** [Azure/simdem#124]
As a user, I am prompted multiple times with 'press a command key to proceed' and have to click enter or another key multiple times to continue execution. Not a big deal, but can be annoying at times to accidently hit enter too many or too few times.

See screen shot for example 
![image](https://user-images.githubusercontent.com/91570951/180574961-a438c821-a8b7-49aa-b8d7-73a9b7652221.png)

**Comment by rgardler-msft** [comment]
Yeah, this one annoys me too. It's surprisingly complex to resolve. But that's because of a need for refactoring in the code. 

Sooner or later we will need to make a call on building a V2. 

For now I will keep chipping away at the code refactoring and if this gets to the point of being fixable I'll throw it in.
